plugins/topology/src/components/Topology/TopologyViewWorkloadComponent.tsx

```diff
@@ -11,6 +11,7 @@ import {
   useVisualizationController,
   VisualizationSurface,
 } from '@patternfly/react-topology';
+import { observer } from 'mobx-react';
```

Comment:
can we use the one from '@patternfly/react-topology' as this also exposes `observer`